### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by YyRrHao** [victoriajx/ORIE-5741-Road-Traffic-Injuries#12]
This project aims to answer how are different attributes and factors could influence a road accident’s severity and try to propose some feasible actions to improve road efficiency and safety.

The dataset that the group used is road traffic injuries in France in 2019, which has more than 58000 accidents.

I like that they properly utilized methods from lectures like one-hot encoding to pre-process the data. The table of data type they listed are clear. There is also visualization included for a direct explanation. They also illustrate the challenge when choosing models, which is imbalanced data and the model neglects the minority part. They also provide solutions for oversampling to solve the problem. The group applied both classification and regression methods regarding the problem, using random forest and multi-class logistic regression respectively. In the final conclusion, the group has provided some really nice advice for governments.

In the multi-class logistic regression result, the test set accuracy is better than the train and validation set. Further explanation or exploration could be done toward that. Also for the multicollinearity, maybe listing out the highly correlated variables could provide a clearer picture. 

In all, the project focuses on a meaningful topic and utilizes various tools to provide a feasible solution.
